api/client/events.go

```diff
@@ -99,7 +100,14 @@ func printOutput(event eventtypes.Message, output io.Writer) {
 
 	if len(event.Actor.Attributes) > 0 {
 		var attrs []string
-		for k, v := range event.Actor.Attributes {
+		var keys []string
+		var v string
```

Comment:
this variable doesn't need to be declared here, use `v := event.Actor.Attributes` in line 110.